src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProvider.java

```diff
@@ -95,4 +96,14 @@ public DefendantPinLetterContentProvider(
         return content;
     }
 
+    private String getPartyNameFor(NamedParty party) {
+        StringBuilder name = new StringBuilder(party.getName());
+
+        if (party instanceof SoleTrader) {
+            (((SoleTrader) party).getBusinessName()).ifPresent(t -> name.append(" T/A ").append(t));
```

Comment:
There's an extra pair of parentheses around `getBusinessName()` that could be removed. Not sure using `StringBuilder` is efficient here but not a big deal.

Reply:
Prefer `t` to be renamed as `businessName`